css/style.css

```diff
@@ -392,4 +394,11 @@ p {
   fill: red;
   stroke: rgb(255, 252, 84);
   transform: scale(0.96, 0.96);
+}
+
+.tab-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
```

Comment:
this looks good but send screenshot so I can see how it looks like

Reply:
![image](https://user-images.githubusercontent.com/79353357/136148083-ef59a3d8-3a9c-449e-9cf1-8d7e0e31d221.png)